paper.md

```diff
@@ -194,11 +194,11 @@ In case of correct specification and absent operator errors, such as mis-typed s
 
 # Adoption and usage
 
-HeuDiConv has [RRID:SCR_017427](https://scicrunch.org/resolver/RRID:SCR_017427) as of time of writing mentions already [6 mentions in papers](https://scicrunch.org/resolver/SCR_017427/mentions?q=&i=rrid:scr_017427).
+As citeable resource [RRID:SCR_017427](https://scicrunch.org/resolver/RRID:SCR_017427), Heudiconv has already [6 mentions in papers](https://scicrunch.org/resolver/SCR_017427/mentions?q=&i=rrid:scr_017427) at time of writing.
 There is a growing number of downloads from PyPI and uses of HeuDiConv (see \autoref{fig:usage}).
-Over 40 BIDS datasets were converted using HeuDiConv with ReproIn heuristic over to BIDS at Dartmouth Brain Imaging Center (DBIC), where ReproIn heuristic being developed.
+Over 40 BIDS datasets were converted over to BIDS with HeuDiConv at Dartmouth Brain Imaging Center (DBIC), using the ReproIn heuristic currently developed there.
```

Comment:
```suggestion
Over 40 BIDS datasets were converted over to BIDS with HeuDiConv at Dartmouth Brain Imaging Center (DBIC), using the ReproIn heuristic developed there.
```
I think it would be easier to swallow this way